crates/satchel-macro/src/lib.rs

```diff
@@ -1,3 +1,8 @@
+//! Procedural attribute macros used by Satchel to register tests and benchmarks.
+//!
+//! These macros mirror Rust's built-in `#[test]` and `#[bench]`, while forwarding metadata such
+//! as `#[should_panic]`, `#[ignore]`, and custom case attributes to the harness via
+//! `satchel::TestCase`.
```

Comment:
```suggestion
//! Procedural attribute macros used by the [satchel](https://docs.rs/satchel/latest/satchel/) crate to register tests and benchmarks.
//!
//! Do not use this crate directly, but through the re-exports in the [satchel](https://docs.rs/satchel/latest/satchel/) crate.
```

For proc-macro crates like this one it is best to warn the users not to use it directly, because they are only supposed to use it via the main satchel crate.

Reply:
done

Reply:
makes a lot of sense :)